src/pages/organize/[orgId]/campaigns/index.tsx

```diff
@@ -33,14 +79,285 @@ export const getServerSideProps : GetServerSideProps = scaffold(async (ctx) => {
     orgId: string;
 };
 
-const AllCampaignsSummaryPage: PageWithLayout<AllCampaignsSummaryPageProps> = () => {
+const AllCampaignsSummaryPage: PageWithLayout<AllCampaignsSummaryPageProps> = ({ orgId }) => {
+    const router = useRouter();
+    const classes = useStyles();
+    const intl = useIntl();
+    const campaignsQuery = useQuery(['campaigns', orgId], getCampaigns(orgId));
+    const upcomingEventsQuery = useQuery(['upcomingEvents', orgId], getUpcomingEvents(orgId));
+    const eventsQuery = useQuery(['events', orgId], getEvents(orgId));
+    const surveysQuery = useQuery(['surveys', orgId], getSurveys(orgId));
+    const callsQuery = useQuery(['calls', orgId], getAllCallAssignments(orgId));
+    const canvassesQuery = useQuery(['canvasses', orgId], getAllCanvassAssignments(orgId));
+
+    const eventMutation = useMutation(postEvent(orgId));
+    const campaignMutation = useMutation(postCampaign(orgId));
+
+    const [CampaignListExpanded, setCampaignListExpanded] = useState(false);
+    const [speedDialOpen, setSpeedDialOpen] = useState(false);
+    const [formDialogOpen, setFormDialogOpen] = useState<null | string>(null);
+    const [filters, setFilters] = useState({
+        callAssignments: false,
+        canvasses: false,
+        standalones: false,
+        surveys: false,
+    });
+    const [cardHeight, setCardHeight] = useState(0);
+    const card = useRef<HTMLDivElement>(null);
+
+    useEffect(() => {
+        setCardHeight(card.current?.offsetHeight || 0);
+    }, []);
+
+    const campaigns = campaignsQuery.data || [];
+
+    const upcomingEvents = upcomingEventsQuery.data || [];
+    const events = eventsQuery.data || [];
+    const surveys = surveysQuery.data?.map(s => ({ data: s, type: 'survey' })) || [];
+    const callAssignments = callsQuery.data?.map(c => ({ data: c, type: 'call_assignment' })) || [];
+    const canvasses = canvassesQuery.data?.map(c => ({ data: c, type: 'canvass_assignment' })) || [];
+    const standalones = events.filter(e => !e.campaign.id).map(s => ({ data: s, type: 'standalone_event' }));
+
+    let unsortedProjects = [
+        ... filters.surveys ? [...surveys] : [],
+        ... filters.canvasses ? [...canvasses] : [],
+        ... filters.callAssignments ? [...callAssignments] : [],
+        ... filters.standalones ? [...standalones] : [],
+    ];
+
+    // if no filters are applied show all unsorted items
+    if (Object.values(filters).every(v => !v)) {
+        unsortedProjects = [...surveys, ...canvasses, ...callAssignments, ...standalones];
+    }
+
+    const handleFilterBoxChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setFilters({ ...filters, [event.target.name]: event.target.checked });
+    };
+
+    const actions = [
+        { icon: <Event />, id: 'event', name: intl.formatMessage({ id: 'misc.speedDial.createEvent' }) },
+        { icon: <People />, id: 'campaign', name: intl.formatMessage({ id: 'misc.speedDial.createCampaign' }) },
```

Comment:
I think this icon is a bit far-fetched for campaign, and could be confusing when we add speed dials to the person database section where people will make more sense.

We've previously used a flag icon for campaigns, which I think works well. Is there something like that? 🚩